Source/WebCore/dom/DocumentMarker.h

```diff
@@ -118,17 +118,16 @@ class DocumentMarker : public CanMakeWeakPtr<DocumentMarker> {
 #endif
 
 #if ENABLE(WRITING_TOOLS)
-    struct UnifiedTextReplacementData {
+    struct WritingToolsTextSuggestionData {
         enum class State: uint8_t {
-            Pending,
-            Committed,
-            Reverted
+            Accepted,
+            Rejected
```

Comment:
Without context about why we have less states now... `enum class State : bool`?

Reply:
I am veryyyy opposed to making enums use `bool` unless they are _actually_ boolean values like No / Yes; it's semantically incorrect, can possibly cause bugs, and offers no real tangible benefit in this case, so I'd prefer to leave as is.